sound/soc/sof/intel/hda.h

```diff
@@ -158,6 +158,13 @@
 #define HDA_DSP_MBOX_UPLINK_OFFSET		0x81000
 
 #define HDA_DSP_STREAM_RESET_TIMEOUT		300
+/*
+ * Timeout in us, for setting the stream RUN bit, during
+ * start/stop the stream. The time calculate the difference
+ * between write the RUN bit, and read the RUN bit to make
+ * sure it is set.
```

Comment:
Suggestion: 
""Timeout in us, for setting the stream RUN bit, during
start/stop the stream. The timeout expires if new RUN bit 
value cannot be read back within the specified time.""

Reply:
OK, thanks!